CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+			is_vassal = no
+			war = no
+			
+			OR = { ruling_party_ideology = communist ruling_party_ideology = communist_social }
+		}		
+		CRO = {	exists = no }
+		
+		
+		has_country_flag = ITA_delle_Chiaie
+    	has_global_flag = ITA_Operation_Tora_Tora
+	}
+
+	mean_time_to_happen = { months = 3 }
+
+	option = {
+		name = "Strike the deal"
+		badboy = 2
+		
+		prestige = 10
+		
+		money = -150000
+		
+		YUG = {
+		    any_owned = {
+			    limit = { is_core = CRO }
+				add_province_modifier = { name = instigating_rebels duration = 3650 }
+			}
+			
+			any_pop = { 
+			    limit = { 
+				    culture = croat
+				    OR = { strata = rich strata = middle } 
+					NOT = { 
+					    OR = { pop_majority_ideology = socialist pop_majority_ideology = communist pop_majority_ideology = communist_social } 
+					} 
+				}
+				move_pop = 749 #Rome
+			} 			
+			any_pop = { 
+			    limit = { culture = croat }
+				scaled_consciousness = { factor = 6 ideology = nationalist }
+				scaled_militancy = { factor = 2 ideology = nationalist }
+				scaled_consciousness = { factor = 6 ideology = populist }
+				scaled_militancy = { factor = 2 ideology = nationalist }
+				ideology = { factor = 0.05 value = nationalist }
+				ideology = { factor = 0.03 value = populist }
+			}
+		}
+		
+		any_pop = { 
+			limit = { culture = croat }
+		    consciousness = -6
+			militancy = -6
+			ideology = { factor = 0.35 value = nationalist }
+		}
+		
+		set_global_flag = ITA_CRO_Ustasha_funding
+		ai_chance = { factor = 0.8 }
+	}
+
+	option = {
+		name = "Reject their offer"
+		prestige = -25
+		
+		any_pop = { 
+			limit = { culture = croat }
+		    consciousness = 2
+			militancy = 2
+		}
+		any_pop = { 
+			limit = { is_primary_culture = yes }
+		    scaled_consciousness = { factor = 2 ideology = nationalist }
+			scaled_militancy = { factor = 2 ideology = nationalist }
+		}
+		
+		ai_chance = { factor = 0.2 }
+	}
+	
+}
+
+#Italy-Croatia Agreement
+country_event = {
+	id = 8020436
+	title = "The Muggia Agreement"
+	desc = EVTDESC8020436
+	picture = "Stefano_delle_Chiaie"
+  
+	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1988
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = populist_dictatorship 
+		war = no
+		is_vassal = no
+		
+		YUG = {
+			exists = yes 
+			
+			OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 } 
+			is_vassal = no
+			war = no
+			
+			OR = { ruling_party_ideology = communist ruling_party_ideology = communist_social }
+		}		
+		CRO = {	exists = no }
+		
+		has_country_flag = ITA_delle_Chiaie
+		has_global_flag = ITA_CRO_Ustasha_funding
+    	has_global_flag = ITA_Operation_Tora_Tora
+	}
+
+	mean_time_to_happen = { months = 2 }
+
+	option = {
+		name = "Istria for Greater Croatia"
+		
+		YUG = {
+		    any_pop = { limit = { culture = croat OR = { type = soldiers type = officers } } move_pop = 771 #Zagreb }
```

Comment:
Please delete #Zagreb as everything past the # in the same line is ignored causing the closing bracket to be ignored.